Library/Homebrew/brew.sh

```diff
@@ -345,7 +345,13 @@ else
   HOMEBREW_OS_USER_AGENT_VERSION="$HOMEBREW_OS_VERSION"
 
   # Ensure the system Curl is a version that supports modern HTTPS certificates.
```

Comment:
Does this apply to 7.25.0 or just 7.41.0?

Reply:
Only 7.41.0.

7.25.0 is just hack to make it work. I'll move the comment.